### DISCUSSION TITLE
How do I open the tree with the file in the current buffer selected/highlighted?

### DISCUSSION THREAD

**Discussion by githorse** [nvim-tree/nvim-tree.lua#2728] [Q&A] [answered]
In [LunarVim](https://www.lunarvim.org/), when I toggle the tree, it opens with the file in the current buffer selected/highlighted. I guess that's not built-in to NvimTree though, since it doesn't seem to happen in the [Clean Room](https://github.com/nvim-tree/nvim-tree.lua/wiki/Troubleshooting#clean-room-replication). (It's not the `path` argument to `:NvimTree{Open,Toggle}`; that changes the root of the tree, which isn't what I want.) 

This may duplicate https://github.com/nvim-tree/nvim-tree.lua/discussions/1929. From reading between the lines on that issue, though, it kind of sounds like NvimTree is supposed to be doing what I want out of the box? Maybe I'm missing something here.

How do I open NvimTree to the file in the current buffer?

**Comment by gegoune** [comment, marked as answer]
`:h nvim-tree-opts-update-focused-file`

**Comment by githorse** [comment]
I don't seem to have that help option (`E149, sorry, no help for nvim-tree-opts-update-focused-file`), but I found it from the top-level NvimTree help. That's definitely the droid I'm looking for! Added this to my setup:

```
require("nvim-tree").setup {
  ...other_stuff,
  update_focused_file = {
    enable = true
  }
}
```

and it works as expected now.

**Reply by alex-courtis** [reply]
Sounds like you need a help reindex: `:helptags ALL`

See `:help helptags`
